test/ng/qSpec.js

```diff
@@ -1586,6 +1598,8 @@ describe('q', function() {
       var rejectedPromise = q.reject('rejected');
       expect(rejectedPromise['finally']).not.toBeUndefined();
       expect(rejectedPromise['catch']).not.toBeUndefined();
+      rejectedPromise.then(noop, noop);
```

Comment:
Seems like you missed a couple of `then(noop, noop)`s here and [there][1].
OOC, is there anything special about them, or just an oversight ?

[1]: https://github.com/angular/angular.js/pull/13662/files#diff-0cfb390955e411a4a5437fa337d52481R2046